src/components/footer/index.tsx

```diff
@@ -7,23 +7,23 @@ import scrtLogo from '@scrtsybil/public/images/scrtnetwork-logo-white.svg';
 
 export default function Footer() {
   const router = useRouter();
-  const routerIsHome = router?.pathname === '/';
+  const routerIsMain = router?.pathname === '/' || router?.pathname === '/learn';
   return (
     <footer
-      className={`z-50 border-t-2 border-gray-800 w-full p-5 flex justify-between items-end bg-black text-white bg-cover font-sans ${
-        routerIsHome ? 'bg-footer-wave' : undefined
+      className={`z-50 border-t-2 border-gray-800 w-full p-5 flex flex-row justify-between bg-black text-white bg-cover font-sans ${
+        routerIsMain ? 'bg-footer-wave items-end' : 'items-center'
       }`}
```

Comment:
hmmm... it seems like you have prettier or eslint overriding our settings - can you look into this and see if you can figure out why? make sure to have these extensions: 
"dbaeumer.vscode-eslint",
  "esbenp.prettier-vscode"

Reply:
I was having a weird issue where prettier is not working without me manually clicking 'format with..' even though it was set to the default formatter and 'format on save' was checked when I check the settings tab (the screen you showed). But I just noticed it was actually set to false in the settings.json for some reason. Now all fixed!

Reply:
umm let me know when you're online so that we can fix this together